streams/src/test/java/org/apache/kafka/streams/state/internals/StoreChangeLoggerTest.java

```diff
@@ -80,6 +86,13 @@ public void testAddRemove() {
 
         changeLogger.logChange(0, null);
         assertNull(logged.get(0));
+    }
+
+    @Test
+    public void testHeadersAreNotSent() {
```

Comment:
nit: rename to `shouldNotSendRecordHeadersToChangelogTopic`

Reply:
ack.